imcsdk/apis/server/adaptor.py

```diff
@@ -13,61 +13,121 @@
 
 
 """
-This module provides apis to setup cisco vic adaptor properties and create vnics
-and vhbas
+This module provides apis to setup cisco vic adaptor properties \
+and create vnics and vhbas
 """
 
+from imcsdk.imccoreutils import get_server_dn
+from imcsdk.imcexception import ImcOperationError
 
-def get_vic_adaptor_properties(handle, adaptor_slot):
+
+def _is_valid_arg(param, kwargs):
+    return param in kwargs and kwargs[param] is not None
+
+
+def _check_and_get_param(param, kwargs):
+    if _is_valid_arg(param, kwargs):
+        return kwargs[param]
+    else:
+        return None
```

Comment:
entire implementation of `_check_and_get_param` could be a one liner while keeping the same behaviour - `return kwargs.get(param)`. I think this returns `None` if param is not found. If that is not the case, it can be modified to return a default value `return kwargs(param, None)`